valid-anagram/GUMUNYEONG.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @param {string} s
+ * @param {string} t
+ * @return {boolean}
+ */
+var isAnagram = function (s, t) {
+    const countHash = {};
+    let result = true;
+
+    if (s.length !== t.length) return false;
+
+    for (str_t of t) {
+        countHash[str_t] ? countHash[str_t]++ : countHash[str_t] = 1;
+    }
+
+    for (str_s of s) {
+        if (countHash[str_s]) {
+            countHash[str_s]--;
+        } else {
+            result = false;
+            break;
+        }
+    }
+
+    return result;
```

Comment:
요렇게 하시면 더 깔끔하고 효율적인 코드가 되지 않을까 생각이 들었습니다 :)

```suggestion
        } else {
            return false;
        }
    }

    return true;
```

Reply:
앗 더 간단하게 표현할 수 있었네요..! 피드백 감사합니다! :>